### ISSUE TITLE
Close all tabs on all browsers

### ISSUE THREAD

**Issue by voidzero** [petko10/real-time-tab-sync#3]
Hi,

Really like your extension. Would you be willing to add "close all tabs on all browsers" as an option? Sometimes the tab merging becomes a bit messy, and you really have to jump through hoops to get the shared tab sessions to essentially clear out. This would be a fantastic option for me.

**Comment by v-ko** [comment]
Yes, this is quite possible. I'm sorry to say I can't promise the next update will come soon, but thanks for the idea :)

**Comment by voidzero** [comment]
Happy to hear you like the idea. For a long while now how Chrome (and Chromium) have been handling tabs, and tab sessions with the browser's "Continue where you left off" setting have been a less than stellar experience, with *and* without the plugin – it's actually how I came across your extension in the first place.

See the thing is, almost every time I open my browser, most of the time tabs I had already closed will be reopened like they were never even closed. So it's like they aren't properly synched or something. I don't know. It sure is annoying. Almost like I just can't get rid of these pages.

And filing bugs with Google's browser team get you nowhere in my experience. So with this function in your addon we would get the best of both worlds: keeping sessions between browsers, and closing them all on all browsers. 

Needless to say, if you can find the time to implement this, I will be very grateful, that's for sure.
